keps/prod-readiness/sig-auth/1205.yaml

```diff
@@ -1,3 +1,3 @@
 kep-number: 1205
-beta:
+stable:
   approver: "@deads2k"
```

Comment:
guessing you should add stable and leave the beta section as-is

Reply:
done